_notices/rgn0028.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: notice
+parent: RAPIDS General Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rgn
+# Update meta-data for notice
+notice_id: 28 # should match notice number
+notice_pin: false # set to true to pin to notice page
+title: "22.06 Hotfix for `cuda-python` Package"
+notice_author: RAPIDS TPM
+notice_status: Completed
+notice_status_color: green
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Release Change
+notice_rapids_version: "v22.06"
+notice_created: 2022-07-07
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-07-07
+---
+
+## Overview
+
+The `cuda-python` package dependency specification was pinned to a maximum version of `11.7.0` for several RAPIDS libraries.
+
+## Rationale
+
+`cuda-python` version `11.7.1` was recently released, which switched its dependency on the [cudatoolkit](https://anaconda.org/nvidia/cudatoolkit) package to the [cuda-toolkit](https://anaconda.org/nvidia/cuda-toolkit) package. This dependency change had unintended consequences for several RAPIDS libraries.
+
+## Impact
+
+All users are recommended to use the latest `22.06.xx` version of all RAPIDS libraries.
```

Comment:
Need to update the version number--believe this is only 22.06 hotfix to date, so should be 22.06.01.

Reply:
updated version to 22.06.01